### DISCUSSION TITLE
Wiki edits don't show on profiles

### DISCUSSION THREAD

**Discussion by hamishmb** [community/community#32883] [Other Feature Feedback, Questions, & Ideas]
Hi there,

Hopefully this is the right place to ask and the correct category.

I have noticed recently that wiki edits don't show as contributions on our profiles. Considering the importance of good documentation, I think it would be good if this could be added :)

Hamish

**Comment by jfullstackdev** [comment]
just want to comment on too, even GitHub Gists are not added up using the Web UI,

but yes, this is a valid request

**Comment by wahitftry** [comment]
Yeap they should add it as contributions too :)

**Comment by citrushouse** [comment]
I just started looking into using GitHub Wikis, and to be honest, I was really surprised they don't count towards the contribution graph. If we have to live with contribution graph anxiety, I'd at least like wiki contributions to count 💀

**Comment by hamishmb** [comment]
I don't think even comments on issues and discussions are counted, on reflection. Agreed - if we're going to have the contribution history, it might as well show all the contributions.
